backend/__tests__/compliance.test.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai';
+import request from 'supertest';
+import app from '../server';
+
+describe('Compliance API', () => {
+  describe('GET /compliance/list', () => {
+    it('should return status 200', async () => {
+      const res = await request(app).get('/compliance/list');
+      expect(res.statusCode).to.equal(200);
+    });
+  });
+});
```

Comment:
This is good unit test but I'd also suggest adding another integration one for checking format of the output.

Reply:
Done